scripts/build-plugins/rollup-plugin-build-themes.js

```diff
@@ -238,7 +244,7 @@ module.exports = function buildThemes(options) {
                     switch (file) {
                         case "index.js": {
                             const isDark = variants[variant].dark;
-                            return `import "${path.resolve(`${location}/theme.css`)}${isDark? "?dark=true": ""}";` +
+                            return `import "${posixPath.resolve(`${location}/theme.css`)}${isDark? "?dark=true": ""}";` +
```

Comment:
This was using the `posix` path implementation before but the real change in this PR is the rest of the `path` usages now use the platform specific `path` implementation so paths from disk resolve correctly.